erpnext/stock/doctype/stock_entry_detail/stock_entry_detail.json

```
@@ -534,11 +539,38 @@
    "label": "Product Recall Notice Item",
    "options": "Product Recall Notice Item",
    "read_only": 1
+  },
+  {
+   "fieldname": "customer",
+   "fieldtype": "Link",
+   "label": "Customer",
+   "options": "Customer",
+   "read_only": 1
+  },
+  {
+   "fieldname": "supplier",
+   "fieldtype": "Link",
+   "label": "Supplier",
+   "options": "Supplier",
+   "read_only": 1
+  },
+  {
+   "fieldname": "section_break_9",
```

```suggestion
   "fieldname": "section_break",
```
keep the fieldnames same